src/user-icon/user-icon.scss

```diff
@@ -92,8 +92,8 @@
 
     &::after {
```

Comment:
Maybe we should include `border-radius` to ensure the icon doesn't appear outside of the status circle shape?

This is what it looks like with & without the `border-radius` and with the mask disabled:
![Screen Shot 2022-03-22 at 5 08 04 PM](https://user-images.githubusercontent.com/849817/159576297-1c50a2ba-a214-43ce-af52-b441ea819bf6.png) ![Screen Shot 2022-03-22 at 5 07 44 PM](https://user-images.githubusercontent.com/849817/159576300-73152044-ed0e-4ca8-a405-70e14cf23e0e.png)